### PR TITLE
test-bot: handle legacy formulae PR from Homebrew/homebrew

### PR DESCRIPTION
**Pull request by xu-cheng**
cc @mikemcquaid

### CODE REVIEW

File: Library/Homebrew/cmd/test-bot.rb [hunk 1]
```diff
@@ -747,6 +752,7 @@ def test_ci_upload(tap)
 
     ARGV << "--verbose"
     ARGV << "--keep-old" if ENV["UPSTREAM_BOTTLE_KEEP_OLD"]
+    ARGV << "--legacy" if ENV{"UPSTREAM_BOTTLE_LEGACY"}
```
Comment (UniqMartin) [inline Library/Homebrew/cmd/test-bot.rb]: Curly braces? I don't think this will work …

Comment (xu-cheng) [inline Library/Homebrew/cmd/test-bot.rb]: thanks for catching up. I have no idea how this comes out. :sweat_smile:

File: Library/Homebrew/cmd/test-bot.rb [hunk 2]
```diff
@@ -11,6 +11,7 @@
 # --junit:         Generate a JUnit XML test results file.
 # --no-bottle:     Run brew install without --build-bottle
 # --keep-old:      Run brew bottle --keep-old to build new bottles for a single platform.
+# --legacy         Bulid formula from legacy Homebrew/homebrew repo.
```
Comment (UniqMartin) [inline Library/Homebrew/cmd/test-bot.rb]: Maybe also add the same comment you made [in `pull.rb`](https://github.com/Homebrew/homebrew/blob/71cba6064f73c3e9ea46d10879f1b503bea86586/Library/Homebrew/cmd/pull.rb#L22):

```
(TODO remove it when it's not longer necessary)
```

**Comment by UniqMartin** [comment]
:+1: LGTM, aside from one more minor comment.

**Pull request closed without merging**